### PR TITLE
Added 'Sample Status' distinction for Sequencing Data Analysis. Sampl…

### PR DESCRIPTION
**Pull request by ajay-p-sharma**
…es that fail Sequencing analysis return status of 'Failed - Illumina Sequencing Analysis'

### CODE REVIEW

File: src/main/java/org/mskcc/limsrest/util/Utils.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
     private final static List<String> CAPTURE_SAMPLE_TYPES = Collections.singletonList("capture library");
     private final static List<String> POOLED_SAMPLE_TYPES = Collections.singletonList("pooled library");
     private final static String FAILED_STATUS_TEXT = "failed";
+    private final static String SEQUENCING_ANALYSIS_STATUS = "Illumina Sequencing Analysis";
```
Comment (DavidStreid) [inline src/main/java/org/mskcc/limsrest/util/Utils.java:45]: Can you use `org.mskcc.limsrest.util.StatusTrackerConfig.STAGE_SEQUENCING_ANALYSIS`?

File: src/main/java/org/mskcc/limsrest/util/Utils.java [hunk 2]
```diff
@@ -490,6 +507,10 @@ private static String getMostAdvancedSampleStatus(DataRecord sample, String requ
                 int currentStatusOrder = getSampleTypeOrder(currentSampleType.toLowerCase());
                 long currentRecordId = current.getRecordId();
                 if (isSequencingComplete(current, user)) {
+                    // Check if Sample has failed Sequencing analysis
+                    if (hasFailedSequencingDataQc(current, user)){
+                        return String.format("%s%s", WORKFLOW_STATUS_FAILED, STAGE_SEQUENCING_ANALYSIS);
```
Comment (DavidStreid) [inline src/main/java/org/mskcc/limsrest/util/Utils.java:512]: `org.mskcc.limsrest.util.StatusTrackerConfig.STAGE_SEQUENCING_ANALYSIS`

**Review by DavidStreid — COMMENTED** [review]
Please change the constant

File: src/main/java/org/mskcc/limsrest/util/Utils.java (hunk 2 above)
Comment (ajay-p-sharma) [reply src/main/java/org/mskcc/limsrest/util/Utils.java:510]: Added the change requested.

**Review by l-wagner — APPROVED** [review]

**Pull request merged**